WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -226,9 +228,15 @@ public boolean onOptionsItemSelected(final MenuItem item) {
 
             if (peopleInviteFragment == null) {
                 Blog blog = WordPress.getCurrentBlog();
-                peopleInviteFragment = PeopleInviteFragment.newInstance(blog.getDotComBlogId());
+                String dotComBlogId = blog.getDotComBlogId();
+                if (dotComBlogId != null && !dotComBlogId.isEmpty()) {
+                    peopleInviteFragment = PeopleInviteFragment.newInstance(dotComBlogId);
+                } else {
+                    AppLog.e(T.PEOPLE, "getDotComBlogId() returned null or empty string!");
```

Comment:
I'll add extra info without checking for `blog` being null since there is code above that will crash if `blog` is null anyway. Thanks!

Having said that and as a note, I think guarding for null `blog` as well would be out of scope of this PR (a case like that would affect many more lines and places than just this one) so, I'm not planning to add such a guard here.